src/components/card/card.js

```diff
@@ -34,41 +31,16 @@ export default function RecipeReviewCard() {
 
   return (
     <div className="cardContainer">
-      <Card className="cardContainer__card">
-        <CardHeader
-          avatar={
-            <Avatar sx={{ bgcolor: red[500] }} aria-label="recipe">
-              R
-            </Avatar>
-          }
-          action={
-            <IconButton aria-label="settings">
-              <MoreVertIcon />
-            </IconButton>
-          }
-          title="Name"
-          subheader="Last Name"
-        />
-        <CardMedia
-          className="cardContainer__studentPhoto"
-          component="img"
-          height="100"
-          width="75"
-          image="https://picsum.photos/id/237/200/300"
-          alt="Student Photo"
-        />
+      <Card>
+        <CardMedia component="img" image={studentAvatar} alt="Student Photo" />
+        <CardHeader title="Full Name" />
+
         <CardContent>
           <Typography variant="body2" color="text.secondary">
             Text with another information of the student.
           </Typography>
         </CardContent>
         <CardActions disableSpacing>
-          <IconButton aria-label="add to favorites">
-            <FavoriteIcon />
-          </IconButton>
-          <IconButton aria-label="share">
-            <ShareIcon />
-          </IconButton>
           <ExpandMore

```

Comment:
this is not the action icon requested in the ticket, please check again the mockup and the ticket to fulfill the requirements